### PR TITLE
Upgrade jetty 8 to 9

### PR DESCRIPTION
**Pull request by yuesong**

### CODE REVIEW

File: hydra-main/src/main/java/com/addthis/hydra/job/web/SpawnService.java [hunk 1]
```diff
@@ -39,93 +39,125 @@
...
-        spawn.getSystemManager().updateSslEnabled(sslEnabled);
+    }
+
+    private SslConnectionFactory createSslConnectionFactory(SpawnServiceConfiguration configuration)
+            throws IOException {
+        SslContextFactory sslContextFactory = new SslContextFactory();
+        sslContextFactory.setKeyStorePath(configuration.keyStorePath);
+        sslContextFactory.setKeyStorePassword(readFile(configuration.keyStorePassword));
+        sslContextFactory.setKeyManagerPassword(readFile(configuration.keyManagerPassword));
+        return new SslConnectionFactory(sslContextFactory, HttpVersion.HTTP_1_1.asString());
```
Comment (jhorwit2) [inline hydra-main/src/main/java/com/addthis/hydra/job/web/SpawnService.java]: Can you do http2?

Comment (yuesong) [reply hydra-main/src/main/java/com/addthis/hydra/job/web/SpawnService.java]: Jetty has http2 support, but I don't know much of it and what that entails for hydra

Comment (jhorwit2) [reply hydra-main/src/main/java/com/addthis/hydra/job/web/SpawnService.java]: It should be invisible for Hydra if you enable it.

Comment (yuesong) [reply hydra-main/src/main/java/com/addthis/hydra/job/web/SpawnService.java]: Turns out that enabling HTTP 2 on the server side is non-trivial amount of work, especially for TLS:

http://www.eclipse.org/jetty/documentation/9.4.8.v20171121/http2.html

Going to leave it as a potential future improvement for now.

**Pull request merged**